cli/urfave/mod_test.go

```diff
@@ -0,0 +1,118 @@
+package urfave
+
+import (
+	"io/ioutil"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	ucli "github.com/urfave/cli/v2"
+	"go.dedis.ch/dela/cli"
+)
+
+func TestBuild(t *testing.T) {
+	builder := NewBuilder("test", nil)
+	app := builder.Build().(*ucli.App)
+
+	app.Writer = ioutil.Discard
+
+	require.Equal(t, "test", app.Name)
+
+	err := app.Run([]string{"test"})
+	require.NoError(t, err)
+}
+
+func TestSetCommand(t *testing.T) {
+	builder := NewBuilder("test", nil)
+
+	builder.SetCommand("first")
+	builder.SetCommand("second")
+
+	app := builder.Build().(*ucli.App)
+
+	// urfave adds by default the -help command. This is why we expect 3.
+	require.Len(t, app.Commands, 3)
```

Comment:
can we partially replace the comment with some asserts about there being a "first", "second" and "help" ? :)